inc/class-wpseo-meta.php

```diff
@@ -411,12 +411,12 @@ public static function sanitize_post_meta( $meta_value, $meta_key ) {
 				break;
 
 
-			case ( $field_def['type'] === 'multiselect' && $meta_key === self::$meta_prefix . 'meta-robots-adv' ):
+			case ( $field_def['type'] === 'hidden' && $meta_key === self::$meta_prefix . 'meta-robots-adv' ):
 				$clean = self::validate_meta_robots_adv( $meta_value );
 				break;
 
 
-			case ( $field_def['type'] === 'url' ):
+			case ( $field_def['type'] === 'url' || $meta_key === self::$meta_prefix . 'canonical' ):
```

Comment:
Which other url input do we have in the meta keys?

Reply:
Now that I think about it: there is still an option for a 'redirect' field.

Alex and I were unable to get it to show up. Do you know under which conditions the redirect field appears? Or is it a leftover from an earlier refactor?

Reply:
Hmm good one. It is supposed to only show up if it has a value already. We discontinued it a couple of years back. But I guess this should be covered by Reactification as well. Of which section is this a field?

Reply:
It should be rendered in the `Advanced Settings` tab. It is defined [here](https://github.com/Yoast/wordpress-seo/blob/18e50f23dbeaa526874f8b131b94004c6fbf5db7/inc/class-wpseo-meta.php#L186).

The only other place where I can find a reference to this field is [here](https://github.com/Yoast/wordpress-seo/blob/18e50f23dbeaa526874f8b131b94004c6fbf5db7/admin/metabox/class-metabox.php#L179) where translations are added.